includes/class-wc-install.php

```diff
@@ -815,10 +819,20 @@ private static function get_schema() {
   permission_id BIGINT UNSIGNED NOT NULL,
   user_id BIGINT UNSIGNED NULL,
   user_ip_address VARCHAR(100) NULL DEFAULT '',
-  PRIMARY KEY (download_log_id),
+  PRIMARY KEY  (download_log_id),
   KEY permission_id (permission_id),
   KEY timestamp (timestamp)
 ) $collate;
+CREATE TABLE {$wpdb->prefix}wc_product_meta_lookup (
+  `product_id` bigint(20) NOT NULL,
+  `min_price` decimal(10,2) NULL default NULL,
+  `max_price` decimal(10,2) NULL default NULL,
+  `average_rating` decimal(3,2) NULL default 0.00,
+  `total_sales` bigint(20) NULL default 0,
+  `stock` bigint(20) NULL default NULL,
+  PRIMARY KEY  (`product_id`),
+  KEY min_max_price (`min_price`, `max_price`)
```

Comment:
Do you have examples of queries that use this index?

Maybe I wasn't clear on my first comment (https://github.com/woocommerce/woocommerce/pull/22718#discussion_r260934689), I wasn't suggesting a change in the order of the fields in the index. I was asking if we needed two indexes, one for queries that sort product by price from low to high and another for queries that sort product by price from high to low.

Reply:
Would that make a difference if all queries sort by both fields?

Go to shop page, sort by price, and apply a price filter. Then open query monitor and grab the main query. Thats what you can use for testing.

Reply:
I tested this an even though it showed up as a possible index to use, it was never used, it used the primary key instead

```

1 | PRIMARY | wp_posts | range | PRIMARY,type_status_date | type_status_date | 164 | NULL | 21 | Using where; Using index; Using temporary; Using f...
-- | -- | -- | -- | -- | -- | -- | -- | -- | --
1 | PRIMARY | wc_product_meta_lookup | eq_ref | PRIMARY,min_max_price | PRIMARY | 8 | wc.wp_posts.ID | 1 | Using where
2 | MATERIALIZED | wp_term_relationships | ref | PRIMARY,term_taxonomy_id | term_taxonomy_id | 8 | const | 3 | Using index
```